thread.c

```diff
@@ -97,13 +97,23 @@ static VALUE rb_cThreadShield;
 static VALUE sym_immediate;
 static VALUE sym_on_blocking;
 static VALUE sym_never;
-static ID id_locals;
 
 enum SLEEP_FLAGS {
-    SLEEP_DEADLOCKABLE = 0x1,
-    SLEEP_SPURIOUS_CHECK = 0x2
+     SLEEP_DEADLOCKABLE = 0x1,
+     SLEEP_SPURIOUS_CHECK = 0x2
```

Comment:
Seems slipping indents.

Reply:
Sorry about that again, just fixed and pushed